### PR TITLE
Remove jasper

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

File: dev-qt/qtimageformats/qtimageformats-5.12.3.ebuild [hunk 1]
```diff
@@ -10,20 +10,18 @@ if [[ ${QT5_BUILD_TYPE} == release ]]; then
...
 	~dev-qt/qtgui-${PV}
 	media-libs/libwebp:=
 	media-libs/tiff:0
-	jpeg2k? ( media-libs/jasper:= )
 	mng? ( media-libs/libmng:= )
 "
 RDEPEND="${DEPEND}"
 
 src_configure() {
-	qt_use_compile_test jpeg2k jasper
```
Comment (SoapGentoo) [inline dev-qt/qtimageformats/qtimageformats-5.12.3.ebuild]: @Pesa how can we safely disable jasper here without triggering automagic?

Comment (Pesa) [reply dev-qt/qtimageformats/qtimageformats-5.12.3.ebuild]: Copy what `qt_use_compile_test` is doing, without the conditional.

But if you're going to remove jasper, there won't be any risk of automagic, right?

Comment (SoapGentoo) [reply dev-qt/qtimageformats/qtimageformats-5.12.3.ebuild]: hence why I'm pondering to just leave it like it is, once jasper is gone

File: profiles/package.mask [hunk 2]
```diff
@@ -29,6 +29,14 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2019-07-14)
+# Unmaintained, removed from Debian and other
```
Comment (mgorny) [inline profiles/package.mask]: What's with the narrow wrapping? You can use up to 80 chars, or 72 if you like ;-P.

Also would be nice to file a central 'treecleaning' bug, so we don't have to go over all of those bugs seeking 'oh no, please don't remove it' comments. Also `Removal in X days`.

**Pull request closed without merging**